src/main/java/com/github/ermay12/regex/Regex.java

```diff
@@ -171,76 +160,13 @@
  *  </tr>
  *  </tbody>
  * </table>
- *
- * <h4>Character Classes</h4>
```

Comment:
Because we're keeping methods that relate to group numbering, we should include this documentation